CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Fixes
 
 - Use fallback if route pattern is MVC ([#1188](https://github.com/getsentry/sentry-dotnet/pull/1188))
+- Make UseSentryTracing discoverable ([#1196](https://github.com/getsentry/sentry-dotnet/issues/1196))
```

Comment:
Danger expects the PR id and not the issue Id that's going to be closed.

```suggestion
- Make UseSentryTracing discoverable ([#1200](https://github.com/getsentry/sentry-dotnet/issues/1200))
```

Reply:
Good to know, thanks!